Signum.React/Scripts/Lines/EntityListBase.tsx

```diff
@@ -68,7 +68,7 @@ export abstract class EntityListBaseController<T extends EntityListBaseProps> ex
       return undefined;
 
     return (
-      <a href="#" className={classes("sf-line-button", "sf-move", btn ? "btn btn-light" : undefined)}
+      <a href="#" className={classes("sf-line-button", "sf-move", "sf-move-step", btn ? "btn btn-light" : undefined)}

```

Comment:
What is this for?

Reply:
sf-move-step is to be able to add css style (in Lines.css) to the EntityRepeater ordering buttons.